src/Analyser/NodeScopeResolver.php

```diff
@@ -2128,6 +2129,21 @@ static function (): void {
 				);
 			}
 
+			if (
+				$functionReflection !== null
+				&& (
+					(in_array($functionReflection->getName(), ['sort', 'rsort'], true) && count($expr->getArgs()) >= 1)
+					|| ($functionReflection->getName() === 'usort' && count($expr->getArgs()) >= 2)
```

Comment:
We don't need to complicate this by checking that the number of args is `>= 2`. Just check that the first arg is always there and join the condition into a single `in_array`.

Reply:
Fixed